db/migrate/20200127171445_add_last_signed_in_at_to_candidate.rb

```diff
@@ -0,0 +1,9 @@
+class AddLastSignedInAtToCandidate < ActiveRecord::Migration[6.0]
+  def up
```

Comment:
`add_column` is a reversible statement, so you can use `change` and remove the `down` method:
 
```suggestion
  def change
```

Reply:
Thanks 👍